data/pages/wiki/syntax.txt

```diff
@@ -437,57 +437,6 @@ When you use the ''%%<code>%%'' or ''%%<file>%%'' syntax as above, you might wan
 
 If you don't want any highlighting but want a downloadable file, specify a dash (''-'') as the language code: ''%%<code - myfile.foo>%%''.
 
-
-===== Embedding HTML and PHP =====
-
-You can embed raw HTML or PHP code into your documents by using the ''%%<html>%%'' or ''%%<php>%%'' tags. (Use uppercase tags if you need to enclose block level elements.)
-
-HTML example:
-
-<code>
-<html>
-This is some <span style="color:red;font-size:150%;">inline HTML</span>
-</html>
-<HTML>
-<p style="border:2px dashed red;">And this is some block HTML</p>
-</HTML>
-</code>
-
-<html>
-This is some <span style="color:red;font-size:150%;">inline HTML</span>
-</html>
-<HTML>
-<p style="border:2px dashed red;">And this is some block HTML</p>
-</HTML>
-
-PHP example:
-
-<code>
-<php>
-echo 'The PHP version: ';
-echo phpversion();
-echo ' (generated inline HTML)';
-</php>
-<PHP>
-echo '<table class="inline"><tr><td>The same, but inside a block level element:</td>';
-echo '<td>'.phpversion().'</td>';
-echo '</tr></table>';
-</PHP>
-</code>
-
-<php>
-echo 'The PHP version: ';
-echo phpversion();
-echo ' (inline HTML)';
-</php>
-<PHP>
-echo '<table class="inline"><tr><td>The same, but inside a block level element:</td>';
-echo '<td>'.phpversion().'</td>';
-echo '</tr></table>';
-</PHP>
-
-**Please Note**: HTML and PHP embedding is disabled by default in the configuration. If disabled, the code is displayed instead of executed.
-
```

Comment:
In the rss section bit lower htmlok is mentioned, already fixed in file in unit tests.